docs/getting-started-guides/docker-multinode/deployDNS.md

```diff
@@ -32,55 +32,38 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 
-## Deploy DNS
+## Deploy DNS on `docker` and `docker-multinode`
 
 ### Get the template file
 
-First of all, download the template dns rc and svc file from
+First of all, download the dns template
 
-[skydns-rc template](skydns-rc.yaml.in)
+[skydns template](skydns.yaml.in)
 
-[skydns-svc template](skydns-svc.yaml.in)
+### Set environment variables
 
-### Set env
+Then you need to set `DNS_REPLICAS`, `DNS_DOMAIN` and `DNS_SERVER_IP` envs
 
-Then you need to set `DNS_REPLICAS` , `DNS_DOMAIN` , `DNS_SERVER_IP` , `KUBE_SERVER` ENV.
-
-```
+```console
 $ export DNS_REPLICAS=1
 
 $ export DNS_DOMAIN=cluster.local # specify in startup parameter `--cluster-domain` for containerized kubelet 
 
 $ export DNS_SERVER_IP=10.0.0.10  # specify in startup parameter `--cluster-dns` for containerized kubelet 
-
-$ export KUBE_SERVER=10.10.103.250 # your master server ip, you may change it
-```
-
-### Replace the corresponding value in the template.
-
-```
-$ sed -e "s/{{ pillar\['dns_replicas'\] }}/${DNS_REPLICAS}/g;s/{{ pillar\['dns_domain'\] }}/${DNS_DOMAIN}/g;s/{kube_server_url}/${KUBE_SERVER}/g;" skydns-rc.yaml.in > ./skydns-rc.yaml
-
-$ sed -e "s/{{ pillar\['dns_server'\] }}/${DNS_SERVER_IP}/g" skydns-svc.yaml.in > ./skydns-svc.yaml
 ```
 
-### Use `kubectl` to create skydns rc and service
-
+### Replace the corresponding value in the template and create the pod
 
-```
-$ kubectl -s "$KUBE_SERVER:8080" --namespace=kube-system create -f ./skydns-rc.yaml
+```console
+$ sed -e "s/{{ pillar\['dns_replicas'\] }}/${DNS_REPLICAS}/g;s/{{ pillar\['dns_domain'\] }}/${DNS_DOMAIN}/g;s/{{ pillar\['dns_server'\] }}/${DNS_SERVER_IP}/g" skydns.yaml.in > ./skydns.yaml
 
-$ kubectl -s "$KUBE_SERVER:8080" --namespace=kube-system create -f ./skydns-svc.yaml
+$ kubectl --namespace=kube-system create -f ./skydns.yaml
```

Comment:
The default is `-s` flag is `http://localhost:8080`, and that's were `apiserver` will listen by default for `docker-multinode` (when running from the same node)
Anyway, it's overkill to specify a separate server option here (the user have to be able to `kubectl get po` for example)

One may use the `-s` flag if running `kubectl` from another node, or `export KUBERNETES_MASTER` for example. But the user should know that